### ISSUE TITLE
Организация конкурса стартапов - задача дададзена [8/1/2017 21:24:33]

### ISSUE THREAD

**Issue by falanster** [diglabby/tasks#20]
Коратка пра ініцыятыву: https://vk.com/minsk_startup_weekend
Патрэба ініцыятывы: Найти стартапы, найти тематические сайты, организационные вопросы на самом мероприятии, ...

**Comment by svetit** [comment]
Добра, што яны да нас звярнуліся, але відавочна, што - не чыталі наш сайт.

Бо ім не патрэбны айці-валанцёры, гэта раз.

У их групе ядзин чэл у якасци кантакта  - Игорь Саевец
Директор Бизнес-инкубатора "С нами будущее"
Я не упэўнена, што яны падыходзяць пад прынцыпы - супольнасць, і некамерцыйнасць

**Comment by falanster** [comment]
так, камерцыйная установа, або установа якая працуе на камерцыю.
+ асаблива не вырашана лічбавая патрэба...
